cfg/docker-compose-prod.yml

```
@@ -79,23 +83,49 @@ services:
     image: otel/opentelemetry-collector-contrib:latest
     container_name: collector
     command: [ "--config=/etc/collector.yml" ]
-    ports:
-      - "4317:4317"
     volumes:
       - ./otel-collector-config.yml:/etc/collector.yml:ro
     depends_on:
-      - clickhouse
+      - tempo
+    restart: always
+  tempo:
+    image: grafana/tempo
+    container_name: tempo
+    command: [ "--config.file=/etc/tempo.yml" ]
+    volumes:
+      - ./tempo.yml:/etc/tempo.yml
+      - tempo_storage:/var/lib/tempo
+    ports:
+      - "3200:3200" # tempo
```

```suggestion
      - "3200:3200"
```

We're under `tempo` already